testProject-8.0/javasource/imagecrop/implementation/ImageUtil.java

```diff
@@ -49,7 +50,8 @@ public static void processImage(IContext context, IMendixObject imageObj,
 		try ( InputStream is = Core.getImage(context, imageObj, false) ) 
 		{
 			BufferedImage originalImage = ImageIO.read(is);
-			BufferedImage alteredImage = new BufferedImage(width, height, BufferedImage.TYPE_INT_RGB);
+			Integer imageType = (originalImage.getColorModel().hasAlpha()) ? BufferedImage.TYPE_INT_ARGB : BufferedImage.TYPE_INT_RGB;
+			BufferedImage alteredImage = new BufferedImage(width, height, imageType;

```

Comment:
Hello, could you add a parentheses at the end of this line?

Reply:
Ah good catch, my bad! Will fix it now

Reply:
Done, I also upped the version number for the project. Should I also do this for the widget itself? Or not needed since no changes in the widget are made?

Reply:
Thanks! Not needed right now, we gonna fix other things to release that.